docs/setup-cluster/deploy-cluster/on-prem/linux-packages.rst

```diff
@@ -135,74 +152,81 @@ Master and Agent
  Configure and Start the Cluster
 *********************************
 
-#. Ensure that an instance of PostgreSQL is running and accessible from the machine where the master
-   will be run.
+.. important::
+
+   Ensure that an instance of PostgreSQL is running and accessible from the machine where the
+   Determined master will run.
 
-#. Edit the :ref:`YAML configuration files <topic-guides_yaml>` at ``/etc/determined/master.yaml``
+-  Edit the :ref:`YAML configuration files <topic-guides_yaml>` at ``/etc/determined/master.yaml``
    (for the master) and ``/etc/determined/agent.yaml`` (for each agent) as appropriate for your
-   setup. Ensure that the user, password, and database name correspond to your PostgreSQL
-   configuration.
+   setup.
 
-   .. code::
+.. important::
 
-      db:
-        host: <PostgreSQL server IP or hostname, e.g., 127.0.0.1 if running on the master>
-        port: <PostgreSQL port, e.g., 5432 by default>
-        name: <Database name, e.g., determined>
-        user: <PostgreSQL user, e.g., postgres>
-        password: <Database password>
+   Ensure that the user, password, and database name correspond to your PostgreSQL configuration.
 
-#. Start the master.
+.. code::
+
+   db:
+     host: <PostgreSQL server IP or hostname, e.g., 127.0.0.1 if running on the master>
+     port: <PostgreSQL port, e.g., 5432 by default>
+     name: <Database name, e.g., determined>
+     user: <PostgreSQL user, e.g., postgres>
+     password: <Database password>
+
+-  Start the master by typing the following command:
 
    .. code::
 
       sudo systemctl start determined-master
 
-   The master can also be run directly with the command ``determined-master``, which may be helpful
-   for experimenting with Determined (e.g., testing different configuration options quickly before
-   writing them to the configuration file).
+   .. note::
+
+      You can also run the master directly using the command ``determined-master``. This may be
+      useful when experimenting with Determined such as when you want to quickly test different
```

Comment:
```suggestion
      useful when experimenting with Determined, such as when you want to quickly test different
```

Reply:
resolved in 
commit 879c5e97b1a92e6ea169fc53aefc69191513c712